src/charm.py

```diff
@@ -94,6 +94,17 @@ def _on_install(self, event: ops.framework.EventBase) -> None:
         process = subprocess.run(cmd, capture_output=True, text=True, check=True, timeout=180)
         logger.debug(f"Command finished. stdout={process.stdout}, " f"stderr={process.stderr}")
 
+    def _on_stop(self, event: ops.StopEvent):
+        """Removes departing unit from the MicroCeph cluster forcefully."""
+        try:
+            microceph.remove_cluster_member(gethostname(), is_force=True)
+        except subprocess.CalledProcessError as e:
+            error_unable_to_connect = f'{gethostname()}?force=1": Unable to connect to:'
```

Comment:
I would trim this message error to simply 'Unable to connect to', based on my experience with catching error messages in Ceph, which tend to change a lot among versions.

Reply:
Well, this is a MicroCluster error string, I wanted to make sure the hostname is also present in the string to make sure that the error is indeed coming for the departing unit (which we just forcefully removed).